README.rst

```diff
@@ -1,10 +1,8 @@
 .. image:: doc/_static/dipy-logo.png
+  :height: 180px
   :target: http://dipy.org
-     :alt: DIPY - Diffusion Imaging in Python
+  :alt: DIPY - Diffusion Imaging in Python
 
-======
```

Comment:
The logo and badge are too close, Can you add a horizontal line to separate them? ` ___`.

Thanks!

Reply:
What about a vertical bar (`|`) to add some vertical space?  9f74e05 uses that. You can view the effect in the online rST editor.

Reply:
ok